inferno-community/uscore-data-sets
# Create testing data set for US Core v5.0.1

# Summary
Add uscore-v501-testing-data-set zip file
Add output folder for future comparison 

# Testing Guidance
Here is steps of testing the zip file:
On Inferno-reference-server:
1. Stop reference server: `docker-compose down`
2. Remove database volume: `docker volume rm inferno-reference-server_fhir-pgdata`
3. Remove database image: `docker image rm inferno-reference-server_db`
4. Rename initdb.sql: `mv initdb.sql initdb.000`
5. Open Dockerfile.database
6. Comment 2nd line in dockerfile.databse `COPY initdb.sql /docker-entrypoint-initdb.d/`
7. Start reference server: `docker-compose up`
8. Go to `http://localhost:8080/reference-server/r4/Pateint`
9. Verify the returned Bundle is empty

On us-core-data-set
10. unzip uscore-testing-data-v501-04-21-2023.zip
11. Open `output/data/d831ec91-c7a3-4a61-9312-7ff0c4a32134.json`
12. `POST` content Bundle resource to `http://localhost:8080/reference-server/r4`
13. Verify no error returned
14. Open `output/data/e91975f5-9445-c11f-cabf-c3c6dae161f2.json`
15. `POST` content Bundle resource to `http://localhost:8080/reference-server/r4`
16. Verify no error returned
17. Go to `http://localhost:8080/reference-server/r4/Pateint`
18. Verify two Patients are returned in Bundle resource. Because the Patient.id are randomly assigned by server, the values may not be 85/355

To restored US Core v5 reference server:
Repeat step 1-3
4. Rename initdb.000: `mv initdb.000 initdb.sql`
5. Open Dockerfile.database
6. Uncomment 2nd line in dockerfile.databse `COPY initdb.sql /docker-entrypoint-initdb.d/`
7. Start reference server: `docker-compose up`
